pkgs/development/python-modules/pyfribidi/default.nix

```
@@ -1,21 +1,23 @@
 { stdenv
 , buildPythonPackage
 , fetchPypi
-, isPy3k
 , isPyPy
+, six
 }:
 
 buildPythonPackage rec {
   version = "0.12.0";
   pname = "pyfribidi";
-  disabled = isPy3k || isPyPy;
+  disabled = isPyPy;
```

```suggestion
  disabled = isPyPy || stdenv.cc.isClang;
```
quick and easy way to disable failing clang builds

---

Thank you! In this case build log looks like there is a single, easy to fix error. Maybe small patch will be enough.